rrfs-test/testinput/rrfs_mpasjedi_2024052700_Ens3Dvar.yaml

```diff
@@ -125,7 +125,7 @@ cost function:
            defer to post: true
 
 test:
-  reference filename: data/ref/rrfs-mpasjedi-ens3dvar.ref
+  reference filename: data/ref/rrfs-mpasjedi-ens3dvar.ref_NEW
```

Comment:
Need to be handled in a similar way

Reply:
The test case data are linked in as `fix/expr_data/mpas_2024052700`. And then the test references are under that as `fix/expr_data/mpas_2024052700/rrfs-mpasjedi-ens3dvar.ref`. For updating the reference data, would you recommend creating a new mpas_2024052700_${timestamp} in RDAS_DATA, and then updating the mpas_2024052700 link to that? Or is there a way to just link to the new reference file (e.g., a link two levels below the main case data link)?

Reply:
@SamuelDegelia-NOAA I created a PR https://github.com/SamuelDegelia-NOAA/RDASApp/pull/1 to your branch.
As the ref/ directory is small in size, we can copy the whole ref/ to ref_20240912/ and update the link inside mpas_2024052700.  Don't worry about duplicate files. Once we implement the `git-mega` tool in the second stage of fix file management, "nominal" duplicate files will actually all be links to a binary file in the 'mega' space. I will try to get this step established in the weekend. Thanks!

Reply:
Okay, I see that you your PR convert `mpas_2024052700` to a directory and then created individual links for each subdirectory. That makes sense to me. I will rebuild and rerun the ctests just to make sure things are working. I'm going to convert this to a draft PR until my testing is done.

Reply:
Generally, we would want to link individual files, not directories, so as to give us the most flexibility.  But by the time I linked `mpas_2024052700`, it did not contain 'ref' so I made a shortcut to only link the top-level directory of mpas_2024052700.  We can gradually get into the leaf of a directory tree whenever needed. Thanks!

Reply:
Yes it would make sense from a tracking perspective to link to individual files. Maybe in a future PR we can incorporate that.

Reply:
> Yes it would make sense from a tracking perspective to link to individual files. Maybe in a future PR we can incorporate that.

Theoretically, it is so. But in practice, we would want to use a **hybrid** method: For example, for crtm data, we always update from one version to another version and hence updating the whole directory directly works better. We would like to avoid tracking those nearly 900 individual files since there is no benefit in tracking those individual files in RDASApp except the negative impact crowding the RDASApp git log history. Similar stories go to `fv3-jedi-data/  ioda-data/  mpas-jedi-data/  ufo-data` etc.  So we can track directories first and when there is a need, we create a directory and then track the subdirectories or files until our needs are met.